bugwarrior/services/bitbucket.py

```diff
@@ -157,12 +173,13 @@ def validate_config(cls, config, target):
         IssueService.validate_config(config, target)
 
     def fetch_issues(self, tag):
-        response = self.get_data('/repositories/%s/issues/' % (tag))
-        return [(tag, issue) for issue in response['values']]
+        response = self.get_collection('/repositories/%s/issues/' % (tag))
+        for issue in response:
+            yield (tag, issue)
```

Comment:
I think I planned to use only iterators but it turned out to require too much changes to the method `issues`. I reverted to a list in the last commit.